src/main/java/com/google/javascript/gents/TypeScriptGenerator.java

```diff
@@ -80,11 +87,12 @@ public static void main(String[] args) {
   private final Options opts;
   private final Compiler compiler;
   private final GentsErrorManager errorManager;
+  private final Map<String, String> externsMap;
 
   final PathUtil pathUtil;
   final NameUtil nameUtil;
 
-  TypeScriptGenerator(Options opts) {
+  TypeScriptGenerator(Options opts) throws FileNotFoundException {
```

Comment:
I'm not sure what you're suggesting? I could do the json parsing in the Options class itself (and then it would be the one throwing FileNotFound), is that what you mean?